lib/orbit_web/controllers/sign_in_export_controller.ex

```diff
@@ -38,7 +38,7 @@ defmodule OrbitWeb.SignInExportController do
         ""
 
       cert ->
-        cert["expires"]
+        cert["expires"] || "missing"
```

Comment:
So a override for a missing cert is stored as an override with `"expires" => nil`? That seems very implicit.

Where it described how this is stored in the database? `operator_sign_in.ex` just has `override: [map()] | nil`. I'm hoping for a struct, typespec, or even just a comment that describes what fields there are in that map, and how overriding missing certifications are represented.

Reply:
Agreed, this is opaque. I added a comment to the [typespec](https://github.com/mbta/orbit/pull/228/files#diff-11f51221667403a857be3385878d97c5f25283655ef63a5a77c4a0cd88fafc6dR23); does that feel satisfying?